### PR TITLE
Final change to move Vessel and SARaircraft to NETN-AIS.

### PR DESCRIPTION
**Pull request by bergtwvd**
Resolves:
https://github.com/AMSP-04/NETN-AIS/issues/14
https://github.com/AMSP-04/NETN-ORG/issues/47

### CODE REVIEW

File: NETN-AIS.xml [hunk 1]
```diff
@@ -638,6 +757,45 @@ This message type is primarily intended for long-range detection of AIS Class A
...
+Both source and destination must represent an AIS station (vessel, SAR aircraft, etc).</semantics>
+                        <parameter>
+                            <name>Entity</name>
+                            <dataType>UuidArrayOfHLAbyte16</dataType>
+                            <semantics>REQUIRED. The entity that represents the destination. This must be an AIS station.</semantics>
+                        </parameter>
+                        <parameter>
+                            <name>Message</name>
+                            <dataType>HLAASCIIstring</dataType>
+                            <semantics>REQUIRED. 1-156 chars of six-bit text.</semantics>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:780]: Perhaps reference some AIS std where this message format is specified?

File: NETN-AIS.xml [hunk 2]
```diff
@@ -638,6 +757,45 @@ This message type is primarily intended for long-range detection of AIS Class A
...
+                        <sharing>PublishSubscribe</sharing>
+                        <transportation>HLAreliable</transportation>
+                        <order>Receive</order>
+                        <semantics>Tasks entity (the source) to send a safety related broadcast message (AIS message type 14).
+
+The source must represent an AIS station (vessel, SAR aircraft, etc).</semantics>
+                        <parameter>
+                            <name>Message</name>
+                            <dataType>HLAASCIIstring</dataType>
+                            <semantics>REQUIRED. 1-161 chars of six-bit text.</semantics>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:794]: Perhaps reference some AIS std where this message format is specified?

File: NETN-AIS.xml [hunk 3]
```diff
@@ -661,7 +819,7 @@ This message type is primarily intended for long-range detection of AIS Class A
                 <units>NA</units>
                 <resolution>NA</resolution>
                 <accuracy>30 bits</accuracy>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:821]: Is the accuracy really 30 bits??? This is an identifier and should always be perfect? What happens if it is not accurate?

File: NETN-AIS.xml [hunk 4]
```diff
@@ -695,6 +853,42 @@ This message type is primarily intended for long-range detection of AIS Class A
                 <accuracy>6 bits</accuracy>
                 <semantics>Functional ID (FID).</semantics>
             </simpleData>
+            <simpleData>
+                <name>IMOType</name>
+                <representation>HLAinteger32BE</representation>
+                <units>NA</units>
+                <resolution>NA</resolution>
+                <accuracy>30 bits</accuracy>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:861]: Is the accuracy really 30 bits??? This is a type indicator and should always be perfect? What happens if it is not accurate?

File: NETN-AIS.xml [hunk 5]
```diff
@@ -695,6 +853,42 @@ This message type is primarily intended for long-range detection of AIS Class A
...
+The IMO number is made of the three letters "IMO" followed by a seven-digit number. This number consists of a six-digit sequential unique number followed by a check digit. The integrity of an IMO number can be verified using its check digit.This is done by multiplying each of the first six digits by a factor of 2 to 7 corresponding to their position from right to left. The rightmost digit of this sum is the check digit. For example, for IMO 9074729: (9×7) + (0×6) + (7×5) + (4×4) + (7×3) + (2×2) = 139. 
+ 
+This attribute represents the 7 digits value of the IMO number. The value shall be zero for not available (default). The value shall also be zero for inland vessels.</semantics>
+            </simpleData>
+            <simpleData>
+                <name>ShipTypeType</name>
+                <representation>HLAoctet</representation>
+                <units>NA</units>
+                <resolution>NA</resolution>
+                <accuracy>8 bits</accuracy>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:873]: Is the accuracy really 8 bits??? This is a type indicator and should always be perfect? What happens if it is not accurate?

File: NETN-AIS.xml (hunk 3 above)
Comment (lofstrandbjorn) [inline NETN-AIS.xml:821]: Is the accuracy really 30 bits??? This is a type indicator and should always be perfect? What happens if it is not accurate?

File: NETN-AIS.xml [hunk 6]
```diff
@@ -695,6 +853,42 @@ This message type is primarily intended for long-range detection of AIS Class A
                 <accuracy>6 bits</accuracy>
```
Comment (lofstrandbjorn) [inline NETN-AIS.xml:853]: Is the accuracy really 6 bits??? This is a type indicator and should always be perfect? What happens if it is not accurate?

File: NETN-AIS.xml (hunk 2 above)
Comment (bergtwvd) [reply NETN-AIS.xml:794]: The standard is referenced in the documentation.

File: NETN-AIS.xml (hunk 3 above)
Comment (bergtwvd) [reply NETN-AIS.xml:821]: The question is more what OMT element I should use to stick in this information. II use accuracy now, but am happy to switch to another element.

File: NETN-AIS.xml (hunk 4 above)
Comment (bergtwvd) [reply NETN-AIS.xml:861]: The question is more what OMT element I should use to stick in this information. II use accuracy now, but am happy to switch to another element.

**Pull request closed without merging**